backend/checkmach.ml

```diff
@@ -25,8 +25,55 @@
  **********************************************************************************)
 [@@@ocaml.warning "+a-30-40-41-42"]
 
+let keep_all_details = ref false
+
 module String = Misc.Stdlib.String
 
+module Detail = struct
+  type context =
```

Comment:
(It looks like only `Somewhere_else`
is actually used.)

Reply:
That's actually by design. The goal is to provide an interface that will likely not change. Some of that information will be computed by #863 and is not available yet.